content/company-info-and-process/communication/content_guidelines/style_and_mechanics.md

```diff
@@ -854,7 +854,7 @@ List of product names:
 - With Sourcegraph Universal Code Search you can...
 - You can search across all your repositories with Universal Code Search.
 - Companies benefit from a universal code search tool because...
-- Sourcegraph Batch Changes are... Batch Changes allow you to...
+- Sourcegraph Batch Changes is... Batch Changes allows you to...
 - To create a batch change...
```

Comment:
@abschu, one potential way to have "batch changes" per your comment:

```suggestion
- To create a batch change...
- You can create batch changes for various purposes, such as upgrading dependencies across...
```

Reply:
I would use
> such as changing an API call site across many repositories